app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+from typing import List, Union
+
+
+def calculate_team_total_rating(players: List[Player]) -> Union[int, float]:
```

Comment:
Use the following type annotation 
```suggestion
def calculate_team_total_rating(players: List[Player]) -> int | float:
```